test/object-mapper.test.ts

```diff
@@ -377,6 +377,53 @@ describe(ObjectMapper.name, () => {
     });
   });
 
+  it(`allows picking properties`, () => {
+    interface Input {
```

Comment:
🙏 Could you please add some more "negative" test scenarios to demonstrate that type safety still applies to the picked items? E.g. `firstName: string | undefined` can't be picked into `firstName: string`. (Doesn't need to be comprehensive, just check that the input property's type must be equivalent to, or a subset of, the output property's type.)

Reply:
☑️ Added negative test scenarios